src/component/project/ProjectList/ProjectList.tsx

```diff
@@ -98,12 +98,12 @@ export const ProjectListNew = () => {
                     className={styles.cardLink}
                 >
                     <ProjectCard
-                        onHover={() => handleHover(project?.id)}
-                        name={project?.name}
-                        memberCount={project?.memberCount ?? 0}
-                        health={project?.health}
-                        id={project?.id}
-                        featureCount={project?.featureCount}
+                        onHover={() => handleHover(project.id)}
+                        name={project.name}
+                        memberCount={project.memberCount ?? 0}
+                        health={project.health}
+                        id={project.id}
+                        featureCount={project.featureCount}
```

Comment:
Oh, this is a nice change. How come we can remove the question marks now? Or where they never necessary?